### PR TITLE
ru: correct translation for index.md

### PR DESCRIPTION
**Pull request by KruASe76**
Current translation is not quite accurate

### CODE REVIEW

**Comment by KruASe76** [comment]
If I have some extra free time, I will PR translation updates for other pages. They all look kinda odd in Russian...
...and Russian translation repo was archived a year ago...

**Comment by ghost** [comment]
Ping @nodejs/nodejs-ru

**Comment by ghost** [comment]
@KruASe76 : Thanks for your 1st contribution.

**Comment by KruASe76** [comment]
> Ping @nodejs/nodejs-ru

as I said before, it was archived a year ago

File: locale/ru/index.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ labels:
   other-downloads: Другие загрузки
   current: Текущая
   lts: LTS
-  tagline-current: Последние возможности
+  tagline-current: Свежие возможности
```
Comment (alexandrtovmach) [inline locale/ru/index.md]:
```suggestion
  tagline-current: Новые возможности
```

File: locale/ru/index.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ labels:
...
   tagline-lts: Рекомендовано для большинства
   changelog: Список изменений
   api: Документация
-  version-schedule-prompt: Или смотрите на
-  version-schedule-prompt-link-text: график LTS
+  version-schedule-prompt: Информацию о поддерживаемых выпусках можно найти на
+  version-schedule-prompt-link-text: графике выпуска
```
Comment (alexandrtovmach) [inline locale/ru/index.md]:
```suggestion
  version-schedule-prompt-link-text: графике выпусков
```

**Review by alexandrtovmach — COMMENTED** [review]
few suggestions

File: locale/ru/index.md (hunk 2 above)
Comment (KruASe76) [reply locale/ru/index.md]: agree

File: locale/ru/index.md (hunk 1 above)
Comment (KruASe76) [reply locale/ru/index.md]: agree, I guess
yesterday I thought "новые" would be better if "new" was in the original text... but now it seems ok to me

**Comment by nschonni** [comment]
> > Ping @nodejs/nodejs-ru
> 
> as I said before, it was archived a year ago

All the old i18n repos were all archived, but the teams still exist and are used for things like this

**Review by alexandrtovmach — APPROVED** [review]
lgtm

**Review by ovflowd — APPROVED** [review]

**Comment by satoshinakamotoVoltage** [comment]
Hi. I came to start.
wwww.mehdimehrabi40277@gmail.com

**Pull request merged**